src/Asset/EntrypointLookupInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Symfony\WebpackEncoreBundle\Asset;
+
+
+interface EntrypointLookupInterface
+{
+    public function getJavaScriptFiles(string $entryName): array;
+
+    public function getCssFiles(string $entryName): array;
+
+    /**
+     * Resets the state of this service.
+     */
+    public function reset();
```

Comment:
TODO for later - when Symfony 4.2 is released, we can make this interface extend https://github.com/symfony/contracts/blob/master/Service/ResetInterface.php